src/AccessibilityInsights.Extensions.GitHub/Properties/Resources.resx

```diff
@@ -159,4 +159,22 @@ This accessibility issue was found using Accessibility Insights for Windows, a t
   <data name="tbURLPlaceHolder" xml:space="preserve">
     <value>Enter desired GitHub repo link</value>
   </data>
+  <data name="AlphaNumricPattern" xml:space="preserve">
+    <value>[a-zA-Z0-9]</value>
```

Comment:
I don't think these need to be in resources because they are essentially programmatic logic in string form, not user-facing strings. But I'm not opposed to keeping them here if you prefer

Reply:
let's keep all the strings in the resource file.